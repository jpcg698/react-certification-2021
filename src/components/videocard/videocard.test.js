import { render, screen } from '@testing-library/react';
import VideoCard from './videocard';
import yt from "../../data/yt"

test('Renders card correctly', () => {

    const Videos = render(<VideoCard 
        thumbnail={yt.items[1].snippet.thumbnails.high.url} 
        title={yt.items[1].snippet.title} url={yt.items[1].id.videoId} 
        description={yt.items[1].snippet.description}
        kind={yt.items[1].id.kind}
        />);

    expect(Videos).toMatchSnapshot();
  });
  