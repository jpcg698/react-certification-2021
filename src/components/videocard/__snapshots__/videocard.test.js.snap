// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders card correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <div
          class="sc-bdnxRM fWlKPq"
        >
          Video Tour | Welcome to Wizeline Guadalajara
        </div>
        <div
          class="sc-bdnxRM fWlKPq"
        >
          <a
            href="https://www.youtube.com/watch?v=nmXMgqjQzls"
          >
            <img
              alt="Follow Hector Padilla, Wizeline Director of Engineering, for a lively tour of our office. In 2018, Wizeline opened its stunning new office in Guadalajara, Jalisco, ..."
              class="sc-gtsrHT hvUsJL"
              src="https://i.ytimg.com/vi/nmXMgqjQzls/hqdefault.jpg"
            />
          </a>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <div
        class="sc-bdnxRM fWlKPq"
      >
        Video Tour | Welcome to Wizeline Guadalajara
      </div>
      <div
        class="sc-bdnxRM fWlKPq"
      >
        <a
          href="https://www.youtube.com/watch?v=nmXMgqjQzls"
        >
          <img
            alt="Follow Hector Padilla, Wizeline Director of Engineering, for a lively tour of our office. In 2018, Wizeline opened its stunning new office in Guadalajara, Jalisco, ..."
            class="sc-gtsrHT hvUsJL"
            src="https://i.ytimg.com/vi/nmXMgqjQzls/hqdefault.jpg"
          />
        </a>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
